src/components/Button/index.tsx

```diff
@@ -58,7 +60,14 @@ const LoaderColor = {
     white: "#85ACB0",
 } as const;
 
+const BorderWidth = {
+    md: "deriv-button__border-width--md",
+    sm: "deriv-button__border-width--sm",
+    none: "deriv-button__border-width--none",
+} as const;
+
 export const Button = ({
+    borderWidth = 'sm',
```

Comment:
```suggestion
    borderWidth = 'md',
```
since the default size is `md` and the border-size for that should be `2px`